src/hw2/Admin.java

```diff
@@ -0,0 +1,14 @@
+package hw2;
+
+class Admin extends Comics {
+    private int id;
+
+    Admin(String a, int b, int i) {
```

Comment:
don't use single letter names for your method parameters, they are not informative